### PR TITLE
[New Rule] Lateral Movement via Startup Folder

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/384

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/windows/lateral_movement_via_startup_folder_rdp_smb.toml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+[metadata]
+creation_date = "2020/10/19"
+ecs_version = ["1.6.0"]
+maturity = "development"
+query_schema_validation = false
```
Comment (bm11100) [inline rules/windows/lateral_movement_via_startup_folder_rdp_smb.toml:5]:
```suggestion

```

File: rules/windows/lateral_movement_via_startup_folder_rdp_smb.toml [hunk 2]
```diff
@@ -0,0 +1,58 @@
+[metadata]
+creation_date = "2020/10/19"
+ecs_version = ["1.6.0"]
+maturity = "development"
```
Comment (bm11100) [inline rules/windows/lateral_movement_via_startup_folder_rdp_smb.toml:4]:
```suggestion
maturity = "production"
```

File: rules/windows/lateral_movement_via_startup_folder_rdp_smb.toml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+query_schema_validation = false
+updated_date = "2020/10/19"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies suspicious file creation on the Startup folder of a remote system. This is a technique that an adversary can
+use to move laterally by dropping a malicious script or executable that will be executed after a reboot or user logon.
```
Comment (bm11100) [inline rules/windows/lateral_movement_via_startup_folder_rdp_smb.toml:12]:
```suggestion
Identifies suspicious file creations in the startup folder of a remote system. An adversary could abuse this
to move laterally by dropping a malicious script or executable that will be executed after a reboot or user logon.
```

Comment (bm11100) [reply rules/windows/lateral_movement_via_startup_folder_rdp_smb.toml:12]: will need to re-lint

**Review by bm11100 — APPROVED** [review]
LGTM 🚀

File: rules/windows/lateral_movement_via_startup_folder_rdp_smb.toml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+references = ["https://www.mdsec.co.uk/2017/06/rdpinception/"]
+risk_score = 73
+rule_id = "25224a80-5a4a-4b8a-991e-6ab390465c4f"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Lateral Movement"]
+type = "eql"
+
+query = '''
+file where event.type in ("creation", "change") and
+ /* via RDP TSClient mounted share or SMB */
```
Comment (seth-goodwin) [inline rules/windows/lateral_movement_via_startup_folder_rdp_smb.toml:26]: I dig in-logic comments 👍

**Review by seth-goodwin — APPROVED** [review]
Looks Good!

**Pull request merged**